src/_scss/pages/covid19/awardSpendingAgency/_awardSpendingAgency.scss

```diff
@@ -12,5 +12,77 @@
 
     .award-spending__content {
         @include display(block);
+
+        .usda-table {
+            margin-top: 0;
+            td {
+                button {
+                    float: left;
+                    margin-top: rem(5);
+                }
+                &:first-child {
+                    display: table-cell;
+                    @include span-columns(6 of 12, table);
+                    text-align: left;
+                    padding-left: rem(15);
+
+                    p {
+                        a {
+                            // IE11 fix for td not having the correct width
+                            @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+                                max-width: 50%;
+                            }
+                        }
+                    }
+                }
+                @include span-columns(2 of 12, table);
+                text-align: right;
+        
+                &:not(:first-child) {
+                    padding-right: rem(40);
+                }
+                .usda-table__child-cell-content {
+                    padding: rem(10) rem(0);
+                }
+            }
+            .table-header {
+                .table-header__label {
+                    @include flex(0 1 auto);
+                    text-align: right;
+                }
+        
+                .table-header__sort {
+                    margin-top: auto;
+                    margin-bottom: auto;
+                }
+        
+                &:not(:first-child) {
+                    .table-header__content {
+                       @include justify-content(flex-end);
+                    }
+                }
+            }
+        }
+        
+        .table-wrapper {
```

Comment:
We should be able to remove this block as of component library v 1.1.4 and #1811